__testUtils/mockAccessRequestDetails.js

```diff
@@ -19,43 +19,43 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-export function getConsentRequestDetailsOnePurpose() {
+export function getAccessRequestDetailsOnePurpose() {
   return {
     "@context": [
       "https://www.w3.org/2018/credentials/v1",
       "https://w3id.org/security/suites/ed25519-2020/v1",
-      "https://consent.pod.inrupt.com/credentials/v1",
+      "https://access.pod.inrupt.com/credentials/v1",
     ],
-    id: "https://consent.pod.inrupt.com/vc/53973727-f4d0-9e8dbdc041fd",
-    type: ["VerifiableCredential", "SolidCredential", "SolidConsentRequest"],
-    issuer: "https://consent.pod.inrupt.com",
+    id: "https://access.pod.inrupt.com/vc/53973727-f4d0-9e8dbdc041fd",
+    type: ["VerifiableCredential", "SolidCredential", "SolidAccessRequest"],
+    issuer: "https://access.pod.inrupt.com",
     issuanceDate: "2021-05-26T16:40:03Z",
     expirationDate: "2022-06-23T16:40:03Z",
     credentialSubject: {
       id: "https://mockappurl.com",
       inbox: "https://mockappurl.com/inbox/",
-      hasConsent: [
+      hasAccess: [
         {
           mode: ["Read", "Write"],
-          hasStatus: "ConsentStatusRequested",
+          hasStatus: "AccessStatusRequested",
           forPersonalData: ["https://pod.inrupt.com/alice/private/data/"],
           forPurpose: "https://example.com/SomeSpecificPurpose",
         },
         {
           mode: ["Read"],
-          hasStatus: "ConsentStatusRequested",
```

Comment:
same as others... in short we only want to change the PB components names and local variables, not anything related to the VC, and that includes mock VCs and the way we destructure/use it in the components